vignettes/git_usage.Rmd

```diff
@@ -55,49 +54,56 @@ You can also create a feature branch in GitHub.
 
 ```{r, echo = FALSE}
-    knitr::include_graphics("GitHub_feature_branch.png", dpi = 144)
+    knitr::include_graphics("github_feature_branch.png", dpi = 144)
 ```
 
 
 ### Commits from the Terminal in RStudio
 
-When committing, prefix the message with the issue number and add a meaningful message  
-`git commit –m '#94 last alive date implementation'`
+To start the commit process, you will need to tell `git` to move your changes to the staging area.  Use `git add <your_file>` to move all changes of `<your_file>` in the staging area to wait for the next commit. You can use `git add .` to move all files you have worked on to the staging area.  Next you can commit, which takes a snapshot of your staged changes.  When committing, prefix the message with the issue number and add a meaningful message `git commit –m '#94 last alive date implementation'`. 
+
+Lastly, you should push your changes up to GitHub using `git push origin <branch name>`
 
 ### Commits from the Git Tab in RStudio
 
-You can also make use of the Git Tab within RStudio to commit your changes. A benefit of using this Tab is being able to see your changes to the file with red and green highlighting. 
+You can also make use of the Git Tab within RStudio to commit your changes. A benefit of using this Tab is being able to see your changes to the file with red and green highlighting. Just like in the terminal, start the message with the issue number and add a meaningful and succinct sentence.  Hit the Commit button and then Push up to GitHub.  
 
 ```{r, echo = FALSE}
-    knitr::include_graphics("GitHub_committ.png", dpi = 144)
+    knitr::include_graphics("github_committ.png", dpi = 144)
 ```
 
 
 **NOTE:**  Placing the issue number in your commit message allows new developers to quickly find discussion surrounding your issue. When pushed to GitHub the issue number will hyperlink to the issue tracker. A powerful tool for discussion.
 
 ## Pull request
 
-We recommend a thorough read through of the articles, [Pull Request Review Guidance](pr_review_guidance.html) and the [Programming Strategy](programming_strategy.html) for in-depth discussions on a doing a proper Pull Request.  Pull Request authors will benefit from shorter review times by closely following the guidance provided in those two articles. Below we discuss some simple `git` commands in the terminal and on GitHub for doing a Pull Request. 
+We recommend a thorough read through of the articles, [Pull Request Review Guidance](pr_review_guidance.html) and the [Programming Strategy](programming_strategy.html) for in-depth discussions on a doing a proper Pull Request.  Pull Request authors will benefit from shorter review times by closely following the guidance provided in those two articles. Below we discuss some simple `git` commands in the terminal and on GitHub for doing a Pull Request. We recommend doing the Pull Request in GitHub only and not through the terminal.  
```

Comment:
Can you even do pull request on the terminal? I thought that way an GitHub exclusive feature...

Reply:
Honestly, I thought you could but have never tried.  https://tighten.co/blog/open-github-pull-request-from-terminal/